Wikipedia/Code/KeychainCredentialsManager.swift

```diff
@@ -0,0 +1,20 @@
+public class KeychainCredentialsManager: NSObject {
+    private var keychainCredentials = WMFKeychainCredentials()
+    @objc public static let shared = KeychainCredentialsManager()
+    
+    @objc public var appInstallID: String? {
```

Comment:
If I wanted to put these inside `WMFAuthenticationManager`, calling `[WMFAuthenticationManager sharedInstance]` in `wmf_applyAppRequestHeaders` (https://github.com/wikimedia/wikipedia-ios/blob/ed91c31bf38a54d72f50540c3acb17e8685b7db6/Wikipedia/Code/AFHTTPRequestSerializer%2BWMFRequestHeaders.m#L12) would crash because `WMFAuthenticationManager` has a reference to `WMFAuthTokenFetcher`, which initializes `AFHTTPSessionManager`